src/main/java/pl/piekoszek/gorskimatches/token/EmailServiceImplemented.java

```diff
@@ -0,0 +1,29 @@
+package pl.piekoszek.gorskimatches.token;
+
+import org.springframework.mail.SimpleMailMessage;
+import org.springframework.stereotype.Component;
+
+import javax.mail.Transport;
+
+
+@Component
+public class EmailServiceImplemented {
```

Comment:
If EmailService configuration class is removed, this can be renamed to `EmailService`

Reply:
ok